RockWeb/Blocks/Finance/ScheduledTransactionSummary.ascx.cs

```diff
@@ -118,7 +118,7 @@ private void ShowContent()
                 FinancialScheduledTransactionService transactionService = new FinancialScheduledTransactionService( rockContext );
 
                 var schedules = transactionService.Queryable( "ScheduledTransactionDetails.Account" )
-                                .Where( s => s.AuthorizedPersonAlias.PersonId == CurrentPerson.Id && s.IsActive == true );
+                                .Where( s => s.AuthorizedPersonAlias.Person.GivingLeaderId == CurrentPerson.GivingLeaderId && s.IsActive == true );
 
```

Comment:
@Tschrock , Nice Catch! Can you switch this to use GivingId though instead of GivingLeaderId. Both are computed columns based on the GivingGroupId value, but GivingLeaderId is using a user-defined function in its computed column and does not have an index. GivingId does not use a function, and has an index (it will be faster).

We have GivingLeaderId for those times you actually need to know who the "leader" is, but in this case you just need to group the people, so GivingId would be better.